readme.md

```diff
@@ -1,72 +1,42 @@
-Updating Retrospect will then automatically be done using the Kodi repository.
-
 # Installing Retrospect #
-There are a couple of ways to install and/or update Retrospect using this GIT repository:
-
-### 1 - Installing and updating Retrospect the 'Easy way' ###
-If a `plugin.video.retrospect-x.x.x.zip` is available from the download section, this zip can be installed using Kodi's _Install from ZIP_ feature. Keep in mind that you these zip files may not always be up-to-date.
-
-This method can also be used to install new versions of Retrospect and thus upgrading older installs.  
-
-### 2 - Installation of Retrospect the 'Advanced way' ###
-The installation comes down to putting the folders from the GIT repo (either via a _'GIT Clone'_ or _'Full Zip Download'_)in the Kodi add-on folder (very often: /home/<user>/.kodi/addons or c:\users\<user>\AppData\Roaming\Kodi\addons). The result should be that the *addons* folder now contains at least these folders:
-
-```
- plugin.video.retrospect
- plugin.video.retrospect.channel.be
- plugin.video.retrospect.channel.mtg
- plugin.video.retrospect.channel.mtv
- plugin.video.retrospect.channel.nick
- plugin.video.retrospect.channel.no
- plugin.video.retrospect.channel.nos
- plugin.video.retrospect.channel.regionalnl
- plugin.video.retrospect.channel.rtlnl
- plugin.video.retrospect.channel.sbsnl
- plugin.video.retrospect.channel.se
- plugin.video.retrospect.channel.streams
- plugin.video.retrospect.channel.uk
- plugin.video.retrospect.channel.videos
-```
-
-#### Option A - ....via 'GIT Clone' ####
-Clone the Nightly GIT repository into a folder of your choice:
-
-```
-git clone https://github.com/retrospect-addon/plugin.video.retrospect.git
-```
-
-The cloned GIT repository should contain the folders mentioned above. Now either `copy` or `symlink` (`junction` on Windows) each those folders into the Kodi Add-ons folder. I would suggest using symlinks or junctions so changes from a `git pull` are automatically available in Kodi. 
-
-#### Option B - ....via 'Full Zip Download' ####
-Download the complete GIT repo and extract it into the Kodi Add-on folder.
+There are two ways to install Retrospect, depending on what version of Kodi you are using:
 
-#### Caution!
-Be aware that if you install it using *Option A* or *Option B* Kodi will **disable** the add-on by default and it will **not install any of the dependencies**. So you need to enable it manually and install all the dependencies by hand. Starting from Kodi Leia this is easier as it has a "View Dependencies" option of add-ons.   
+### Kodi Leia and later
+Let's start with the most recent version of Kodi: Leia (18.0) and newer version such as Matrix (19.0). Users running these version of Kodi can easy install Retrospect via the Kodi repository. Follow these steps (keep in mind they might slightly differ depending on your skin):
```

Comment:
Keep it simple. Rather than 
> Let's start with the most recent version...

simply state:
> Starting from Kodi Leia (v18), you can install Retrospect from within Kodi. Simply search the add-ons in Kodi for *"Retrospect"* and install it.

This won't require an update when v19 is released and doesn't make things more complicated than it should be. This is something people can remember easily.

Reply:
If you still think some people need better/more guidance, I would add a reference to the Wiki with more details:

> More information about [installing Retrospect are available from the Wiki](https://github.com/retrospect-kodi/plugin.video.retrospect/wiki/Installing).